.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!-- 
+Thank you for supporting us with your Pull Request! 🙌 ❤️  
+Before submitting, please take the time to check the points below and provide some descriptive information.
+-->
+
+## Checklist
+
+* [ ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
+* [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
+* [ ] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes #41)
```

Comment:
I would say 'Set a title' sounds a bit better

Reply:
But the focus here is not on setting a title, but choosing one that is descriptive. So that should in some way be reflected.